File: Arrays/1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution 
{
    public boolean check(int[] nums) 
    {
        int ctr = 0;
        for(int i=0; i<nums.length; i++)
        {
            if(nums[i] > nums[(i+1)%nums.length])
            {
                ctr++;
            }
            if(ctr>1)
            {
                return false;
            }
        }   
        return true;
    }
}

File: Arrays/1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        
        size = len(nums)
        
        counter = 0

        for i in range(0, size):

            if nums[i] > nums[(i+1)%size]:
                counter += 1

            if counter > 1:
                return False

        return True 

